srcpkgs/raft/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'raft'
+pkgname=raft
+version=0.9.16
+revision=1
+build_style=gnu-configure
+configure_args="--enable-example=no"
+hostmakedepends="pkg-config autoconf automake libtool"
```

Comment:
`autoconf` is redundant, anyway it's harmless

Reply:
fixed it for one template, forgot for the other. not worth the CPU for another CI round IMO. we have a planet to save, after all.